src/dns/azext_dns/__init__.py

```diff
@@ -1,30 +0,0 @@
-# --------------------------------------------------------------------------------------------
```

Comment:
I don't think it's ok to delete the folder even if it's not be updated anymore.

Reply:
we can delete it in our CI system by running some scripts. But I don't think it's good to delete it at all.